httpserver/lmq_server.cpp

```
@@ -130,11 +127,12 @@ void OxenmqServer::handle_onion_request(oxenmq::Message& message, bool v2) {
         return;
     }
 
-    const auto& eph_key = message.data[0];
+    auto eph_key = extract_x25519_from_hex(message.data[0]);
+    if (!eph_key) return;

```

ditto about message handlers with silent failure being no bueno.